library/package.json

```diff
@@ -125,6 +135,7 @@
   "scripts": {
     "test": "node ../scripts/run-tap.js",
     "test:ci": "CI=true node ../scripts/run-tap.js",
+    "test:new": "node ../scripts/run-tap.js --test-new-instrumentation",
```

Comment:
let's add a list of tests that are passing right now? So that we can run them as part of CI? but the remaining ones should be allowed to fail?